requirements/test.txt

```diff
@@ -4,3 +4,4 @@
 
 pytest>=5
 pytest-cov>=2
+testbook
```

Comment:
It looks like the CPU test is failing because `tritonclient` is not installed. Perhaps we should add `tritonclient` here in `requirements/test.txt`?

Reply:
Thanks @edknv. Based on the discussion yesterday, I will remove the code script for serving nb from the unit test file so it should work now.